internal/dag/gatewayapi_processor.go

```diff
@@ -66,11 +70,34 @@ func (p *GatewayAPIProcessor) Run(dag *DAG, source *KubernetesCache) {
 		p.source = nil
 	}()
 
-	// Gateway must be defined for resources to be processed.
+	// Gateway and GAtewayCLass must be defined for resources to be processed.
```

Comment:
nit: s/GAtewayCLass/GatewayClass

(Unless my mac is rendering wrong again)
![image](https://user-images.githubusercontent.com/1048184/122946486-3e2c9e80-d32e-11eb-9adb-e75347210c7c.png)